peer/main.go

```diff
@@ -253,15 +253,22 @@ func main() {
 	viper.BindPFlag("peer.gomaxprocs", flags.Lookup("peer-gomaxprocs"))
 	viper.BindPFlag("peer.discovery.enabled", flags.Lookup("peer-discovery-enabled"))
 
+	var alternativeCfgPath = os.Getenv("PEER_CFG_PATH")
```

Comment:
This part can be removed if we do not need modifiable configuration file locations.